Merge 1.7.1 into 1.7 branch. (#547)
elastic/ecctl: docs/release_notes/v1.7.1.adoc

[id="{p}-release-notes-v1.7.1"]
=== Elastic Cloud Control (ecctl) v1.7.1
++++
<titleabbrev>v1.7.1</titleabbrev>
++++

> <<{p}-release-notes-v1.7.1-whats-new,What's new>> | <<{p}-release-notes-v1.7.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.1-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.1-changelog,Changelog>>

Reviewer: ```suggestion
<<{p}-release-notes-v1.7.1-changelog,Changelog>>
```
Other refs does not exist, hence the error

[id="{p}-release-notes-v1.7.1"]
=== Elastic Cloud Control (ecctl) v1.7.1
++++
<titleabbrev>v1.7.1</titleabbrev>
++++

<<{p}-release-notes-v1.7.1-whats-new,What's new>> | <<{p}-release-notes-v1.7.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.1-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.1-changelog,Changelog>>

> Welcome to the v1.7.1 release of {n}. This version brings new features, some breaking changes, and bug fixes.

Reviewer: ```suggestion
Welcome to the v1.7.1 release of {n}. This version brings a bug fix.
```
Feel free to adjust this sentence explaining the purpose of this release

Reviewer: I think you just edited my suggestion but did not committed it 😉

Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).

Download the release binaries:

[%hardbreaks]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_darwin_amd64.tar.gz[ecctl_1.7.1_darwin_amd64.tar.gz]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_32-bit.deb[ecctl_1.7.1_linux_32-bit.deb]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_32-bit.rpm[ecctl_1.7.1_linux_32-bit.rpm]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_386.tar.gz[ecctl_1.7.1_linux_386.tar.gz]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_64-bit.deb[ecctl_1.7.1_linux_64-bit.deb]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_64-bit.rpm[ecctl_1.7.1_linux_64-bit.rpm]
https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_amd64.tar.gz[ecctl_1.7.1_linux_amd64.tar.gz]

[float]
[id="{p}-release-notes-v1.7.1-whats-new"]
==== What's new

> // TODO: FILL OR REMOVE

Reviewer: I think you need to actually "fill or remove" this :) Maybe remove this

Author: Hah! I was juggling a bunch of stuff when I made this and completely missed this. Thanks!

[float]
[id="{p}-release-notes-v1.7.1-bug-fixes"]
==== Bug fixes

> // TODO: FILL OR REMOVE

Reviewer: I "think" this is where you should tell what we are doing here (it's kinda bugfix? )

[float]
[id="{p}-release-notes-v1.7.1-breaking-changes"]
==== Breaking changes

> // TODO: FILL OR REMOVE

Reviewer: Remove this section
